src/styles/parts/button.sass

```diff
@@ -0,0 +1,20 @@
+.button
+	display: inline-block
+	margin: 0.25em
+	border: 1px solid
+	padding: 0.5em
+	text-transform: uppercase
```

Comment:
Prosím bez tohoto typografického pekla.

Reply:
`uppercase` se často používá v designech u tlačítek. Proto jsem ho zvolil i zde, aby bylo jasnější, že to jsou tlačítka.